dev-utils/poly-cli/src/templates/manifest.js

```diff
@@ -10,10 +10,15 @@ export function manifestTemplate(feature_name, author) {
             de: {
                 author: "",
                 description: "",
-                links: {},
             },
         },
     };
 
-    return JSON.stringify(t, null, 2);
+    if (feature_type === "preview") {
```

Comment:
A switch/case statement would probably be still a good option; anyway, since @GhenadiePusca has suggested the same thing in several cases, many switch/case statements with the same options call for using... Abstraction! In the shape of a base class, and derived classes that override the specific parts they need. As we will need at least another option, maybe this is the time to think about (and maybe act upon) that